--- src/CMakeLists.txt
+++ src/CMakeLists.txt
@@ -128,7 +128,7 @@
 	target_link_options(CemuBin PRIVATE -z noexecstack)
 	# some residual debug info from boost/discord-rpc is normally included
 	# most likely not helpful in debugging problems with cemu code
-	target_link_options(CemuBin PRIVATE "$<$<CONFIG:Release>:-Xlinker;--strip-debug>")
+	target_link_options(CemuBin PRIVATE "$<$<CONFIG:Release>:-Xlinker;>")
 endif()
 
 if (ENABLE_WXWIDGETS)
